### ISSUE TITLE
Query dimension date

### ISSUE THREAD

**Issue by iamulum** [rongfengliang/cubejs-trino-driver#1]
hi @rongfengliang,
Currently, I'm using your driver here for my trino project.
And when I perform query filter date just like an example in cube official site [here](https://cube.dev/docs/query-format#filters-operators-before-date), I got an error like this
```
{"error":"Error: Unknown timezone: undefined"}
```
Is there a reason for this commit f0d9dcbdbd102f51695fac1a126e76377b40c67f?
Because I have to revert these changes to perform query date.

Or is there any different approach to perform `query dimension type date` using this driver?

**Comment by rongfengliang** [comment]
@iamulum  Yes.  but I think you can do date type query. I disable this because I do some test cube.js will use timezone. but
current cube.js's  core SQL query not  works well with trino timezone . I will  try to fix this for support timezone

**Comment by iamulum** [comment]
> @iamulum Yes. but I think you can do date type query. I disable this because I do some test cube.js will use timezone. but current cube.js's core SQL query not works well with trino timezone . I will try to fix this for support timezone

For my short-term solution here, I've forked your driver and reverted changes in that commit. But I'm not sure it's the best solution for your test case.

I hope to see a better solution to this problem. Thanks for the support 😁

**Comment by rongfengliang** [comment]
@iamulum  I will  fix it for better support trino

**Comment by kodeine** [comment]
Hello,
I started using this as well and face same issue. Are there any further updates to support trino better?

**Comment by rongfengliang** [comment]
@kodeine  I add timezone support .but currently  not well tested you can have a try
